Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -363,6 +379,13 @@ export async function getBundleIdFromPlist(plistPath: string) {
     return bundleId;
 }
 
+function isMacOSProfile(provProfilePath: string): boolean {
+    const result = provProfilePath.endsWith('.provisionprofile');
```

Comment:
I'm confused by this test: I thought all provision profiles had the extension `.provisionprofile`.  Clearly, I am mistaken.  What other values might you expect from this?

Separate question: is case-sensitivity an issue with this test?

Reply:
NM, I see in the tests below about a file called `.mobileprovision`.  My mistake.

Reply:
What do you think about adding a comment here along the lines of: `// .mobileprovision are used for iOS, tvOS, and watchOS and .provisionprofile are used for macOS`?

Reply:
We need to handle case-sensitivity

Reply:
Adding toLowerCase(), though it would be odd for Xcode to switch the case of embedded.mobileprovision.